src/components/Paragraph/index.svelte.d.ts

```diff
@@ -0,0 +1,18 @@
+import { SvelteComponentTyped } from 'svelte';
+export default class Index extends SvelteComponentTyped<IndexProps, IndexEvents, IndexSlots> {
+}
+declare const propDef: {
+    props?: {
+        class?: string;
+    };
+    events?: {
+        [evt: string]: CustomEvent<any>;
+    };
+    slots?: {
+        default: {};
+    };
+};
```

Comment:
how does it work? What's the idea here? please leave a inline comment

Reply:
@zolotokrylin These are auto-generated types definitions.